messages/crypto.json

```diff
@@ -0,0 +1,3 @@
+{
+  "MacKeychainOutOfSync": "\nWe’ve encountered an error with the Mac keychain being out of sync with your `sfdx` credentials. To fix the problem, sync your credentials by logging into your Dev Hub using the auth commands."
```

Comment:
I was about to merge this @williamruemmele-sf but then I noticed it said `by logging into your Dev Hub using...`. This can fail with any org though right? Maybe we should say, `by authenticating into your org again using...` or something like that?

Reply:
yeah, I think 'authenticating' and 'logging in' are both conveying the same action - I can change that